.jshintrc

```diff
@@ -11,11 +11,7 @@
   "newcap": true,
   "noarg": true,
   "quotmark": "single",
-  "regexp": true,
   "undef": true,
   "unused": true,
-  "strict": true,
-  "trailing": true,
-  "smarttabs": true,
-  "white": true
+  "strict": true
```

Comment:
Yeah, well, I didn't care to look into how we might write a Yeoman generator in ES6. In theory, I agree. :)